city/models.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import Column, Integer, String, Text
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class City(Base):
+    __tablename__ = "city"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(100), nullable=False, unique=True)
+    additional_info = Column(Text, nullable=True)
+
+    # temperature = relationship("Temperature", back_populates="city")
```

Comment:
The relationship to the `Temperature` model is commented out. If you intend to establish a bidirectional relationship between `City` and `Temperature`, consider uncommenting this line and ensuring that the `Temperature` model has a corresponding `back_populates` attribute. This will allow you to navigate the relationship from `City` to `Temperature` instances.